adafruit_bitmap_font/pcf.py

```diff
@@ -260,7 +266,7 @@ def _read_accelerator_tables(self):
             ink_maxbounds,
         )
 
-    def _read_properties(self):
+    def _read_properties(self) -> Generator[tuple, None, None]:
```

Comment:
Since this `Generator` only needs to define the yield type, we can actually import `Iterator` from `typing` and use that:

```suggestion
    def _read_properties(self) -> Iterator[Tuple[bytes, Union[bytes, int]]]:
```